csufmse/student_information_system
# auto-assign section numbers

Testing: 
* find a course that has a section in a (open = current) semester.
* Click Add Section, fill out
* Save, notice sec num is one greater.
* Find a course in a current sem with NO section
* aDd a sec, notice number = 1
* Profit!

Fixes #166.
